component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java

```diff
@@ -188,13 +189,22 @@ public static ProfileEntity buildEntityProfile(Profile profile, String restPath,
     userEntity.setAvatar(profile.getAvatarUrl());
     userEntity.setBanner(profile.getBannerUrl());
     try {
+      boolean isExternal = profile.getProperty(Profile.EXTERNAL) != null && (profile.getProperty(Profile.EXTERNAL)).equals("true");
       OrganizationService organizationService = getOrganizationService();
       User user = organizationService.getUserHandler().findUserByName(userEntity.getUsername());
       if (user != null) {
         userEntity.setIsInternal(user.isInternalStore());
+        if (isExternal && user.getLastLoginTime().equals(user.getCreatedDate())) {
+          userEntity.setLastConnexion("neverConnected");
+        } else if (user.getLastLoginTime().equals(user.getCreatedDate())) {
+          userEntity.setLastConnexion("invitedToJoin");
+        } else {
+          String lastLogin = TimeConvertUtils.getFormatDate(user.getLastLoginTime(), "dd/MM/yyyy HH:mm");
```

Comment:
I think it's better to use this reusable component to be able to display date with adequate Date Format switch user language:
https://github.com/Meeds-io/social/blob/develop/webapp/portlet/src/main/webapp/common/components/DateFormat.vue
@azayati